README.md

```diff
@@ -21,14 +23,48 @@ npm install @tbd54566975/dwn-sdk-js
 
 [API docs](https://tbd54566975.github.io/dwn-sdk-js/)
 
+Some projects that use this library: 
+
+* [Example CLI](https://github.com/TBD54566975/dwn-cli)
+* [Example with a web wallet](https://github.com/TBD54566975/incubating-web5-labs/)
+* [Server side aggregator](https://github.com/TBD54566975/dwn-server)
+
 ```javascript
-import { Dwn } from '@tbd54566975/dwn-sdk-js';
 
-// cool things
+import { Dwn, RecordsWrite, RecordsQuery } from '@tbd54566975/dwn-sdk-js';
+
+export const dwn = await Dwn.create({});
+
+....
+const query = await RecordsWrite.create({
+  data                        : randomBytes(randomInt(50, 500)),
+  dataFormat                  : 'application/json',
+  published                   : true,
+  protocol                    : 'yeeter',
+  schema                      : 'yeeter/post',
+  authorizationSignatureInput : signatureMaterial
```

Comment:
might be nice to include easiest way to generate did + `signatureMaterial` since both are basically necessary prerequisites to using dwns

Reply:
thoughts? unrelated to this PR specifically, but it might be helpful to introduce utilities to this sdk so that people have everything they need to use DWNs.

Reply:
yeah I think it is lacking without it, so perhaps the readme should just point to example apps that use it instead?

Reply:
@michaelneale, I think the intent is useful, let me do a refactor then update the README to meet @mistermoe's suggestion.

Reply:
Also, let me know if you think `SignatureMaterial` makes more sense than `SignatureInput`, I can do an opportunistic rename. As a convenience reminder:

```typescript
export type SignatureInput = {
  protectedHeader: JwsHeaderParameters
  privateJwk: PrivateJwk
};
```

Reply:
Done.